tests/settings.py

```diff
@@ -16,6 +16,7 @@
     'django.contrib.staticfiles',
     'openwisp_utils.admin_theme',
     'django.contrib.sites',
+    'rest_framework',
```

Comment:
@nemesisdesign This is for new api to work in the browser (with all those fancy drf html pages). This is not vitally important, we can remove this if you don't want it here.

Reply:
why don't you just do it with a simple json view from django? If we add this here every project using openwisp-utils will have to download DRF which is definitely overkill